src/test/java/com/example/rets_api/repository/UtilsTest.java

```diff
@@ -50,9 +56,20 @@ public static RoomEntity createRoomEntity(RoomType roomType){
         room.setAreaUnit(AreaUnit.SQ_METERS);
         room.setAreaType(AreaType.FLOAT);
         room.setBathSize(BathSize.THREE_QUARTER);
+        room.setIndicator(Indicator.YES);
         return room;
     }
 
+    public static AnimalPolicyEntity createAnimalPolicy(String permittedType){
+        AnimalPolicyEntity animalPolicy = new AnimalPolicyEntity();
+        animalPolicy.setAnimalsPermitted(Indicator.YES);
+        animalPolicy.setPermittedTypes(permittedType);
+        animalPolicy.setWeightUnit(WeightUnit.KILO);
+        animalPolicy.setWeightLimit(10L);
+        animalPolicy.setProperties(Collections.singletonList(createPropertyEntityWithBasicFields()));
```

Comment:
As we are using this list to test, is good to use asList instead Collections.singletonList, because if in the future we need to change this list we'll not have error like UnsupportedOperationException... Collections.singletonList will generate an immutable list.